recipe/meta.yaml

```
@@ -41,25 +44,36 @@ build:
   skip: true  # [win]
   skip: true  # [py<35]
   detect_binary_files_with_prefix: true
+  {% if mpi != 'nompi' %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
 
   # add build string so packages can depend on
   # dagmc or nodagmc variants explicitly:
   # `openmc * nodagmc_*` for openmc without dagmc
   # `openmc * dagmc_*` for openmc with dagmc
+  # mpi or nompi variants explicitly:
+  # `openmc * mpi_mpich_*` for mpich
+  # `openmc * mpi_*` for any mpi
+  # `openmc * nompi_*` for no mpi
 
-  string: {{ dagmc }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ build }}
+  string: {{ dagmc }}_{{ mpi_prefix }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ build }}
 
 requirements:
   build:
     - cmake
     - make
-    - dagmc 3.2.1  # [dagmc == 'dagmc']
+    - dagmc * 3.2.1 * {{ mpi_prefix }}_*  # [dagmc == 'dagmc']
```

should this dagmc line be the same as the dagmc line in the host section
```suggestion
    - dagmc 3.2.1 {{ mpi_prefix }}_*  # [dagmc == 'dagmc']
```